shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -356,16 +370,19 @@ class JSBackend(pgrm: Pgrm) {
 
   def translateClassMember(
       method: MethodDef[Left[Term, Type]]
-  ): JSClassMemberDecl = {
+  )(implicit scope: Scope): JSClassMemberDecl = {
     val name = method.nme.name
     method.rhs.value match {
-      case Lam(Var(param), rhs)         => JSClassMethod(name, JSNamePattern(param) :: Nil, L(translateTerm(rhs)))
-      case Lam(Asc(Var(param), _), rhs) => JSClassMethod(name, JSNamePattern(param) :: Nil, L(translateTerm(rhs)))
-      case Lam(Tup(params), body) => JSClassMethod(
-        name,
-        params map { case _ -> param => translatePattern(param) },
-        L(translateTerm(body))
-      )
+      case Lam(Var(param), rhs) =>
```

Comment:
Does this case really occur? I thought we were always wrapping parameter lists in tuples now?

Reply:
I just realized parameters are always wrapped in tuples. I will remove it.